src/TraceEvent/TraceEventNativeMethods.cs

```diff
@@ -499,33 +588,93 @@ internal unsafe struct EVENT_FILTER_EVENT_ID
 
         [DllImport("advapi32.dll", CharSet = CharSet.Unicode)]
         internal static extern int StartTraceW(
```

Comment:
Any reason that all the pinvokes that have helpers here can't be private?

Reply:
Will fix (where possible).